src/scripts.js

```diff
@@ -143,6 +143,35 @@ function dropYear(dates) {
     }
   });
 
+  //User Hydration Input
+  $('.hydration-submit').click(hydrationHandler);
+
+  function hydrationHandler() {
+    let dateInput = $('#user-hydration-date').val();
+    let ounceInput = Number($('#user-hydration-oz').val());
+
+    fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/hydration/hydrationData', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({
+        userID: user.id,
+        date: `${dateInput}`,
+        numOunces: ounceInput
+      })
+    })
+    .then($('.hydration-submit').after('<p class="success-message">Data Submitted Successfully!</p>'))
+    .catch();
```

Comment:
I wonder if this is where we will add our error states and put more functionality like that in a then block or catch block